.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,10 @@
+#Thu Nov 13 13:26:19 EST 2008
+eclipse.preferences.version=1
```

Comment:
Please remove eclipse specific settings from the PR. Also, please add .settings and .project in .gitignore so as to avoid including eclipse settings in the PR.